addon/index.js

```diff
@@ -45,6 +44,8 @@ const WebExtensionChannels = require('./lib/webextension-channels');
 const ToolbarButton = require('./lib/toolbar-button');
 const ExperimentNotifications = require('./lib/experiment-notifications');
 
+const EXPERIMENT_UPDATE_INTERVAL = 24 * 60 * 60 * 1000; // 1 day
```

Comment:
I think this is going to need a rebase against master, [because we changed this to 12 hours in the notifications PR](https://github.com/mozilla/testpilot/blob/master/addon/index.js#L10). But, the `const` got moved to the top, so this won't technically look like a conflict.